api/vitest.config.ts

```diff
@@ -0,0 +1,10 @@
+import { defineConfig } from 'vitest/config';
```

Comment:
this file is duplicated and throws an error when both files exist, this one should be removed in favour of the file in `api/src/vitest.config.ts` I think?